app/buyers/views/buyers.py

```diff
@@ -298,19 +298,18 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
 
     brief_responses = data_api_client.find_brief_responses(brief_id)['briefResponses']
 
-    met = None
+    new_flow_brief = (datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
```

Comment:
I might suggest something like 'brief_response_has_evidence' so instead of using something like new/old/legacy we have something that instead describes what is different about the brief response and it's data.